quotas.go

```diff
@@ -0,0 +1,36 @@
+package globalsign
+
+import (
+	"net/http"
+)
+
+// QuotasResponse GlobalSign API response of `/quotas/signatures` and `/quotas/timestamps` endpoint.
+type QuotasResponse struct {
+	Value int `json:"value"`
```

Comment:
does this have any units? signatures per some time period?

Reply:
Not having a unit for this one, only return a number of quotas as an integer.